tests/providers/amazon/aws/transfers/test_gcs_to_s3.py

```diff
@@ -57,7 +57,7 @@ def test_execute__match_glob(self, mock_hook):
 
             operator = GCSToS3Operator(
                 task_id=TASK_ID,
-                bucket=GCS_BUCKET,
+                gcs_bucket=GCS_BUCKET,
```

Comment:
All unit tests (except for the one that checks for backwards compatibility) use the updated naming.

Reply:
Nice :)